rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapStorageProviderImpl.java

```diff
@@ -24,7 +24,7 @@ public Optional<Coin> getLockingCap(ActivationConfig.ForBlock activations) {
             if (lockingCap == null) {
                 initializeLockingCap();
             }
-            return Optional.of(lockingCap);
+            return Optional.ofNullable(lockingCap);
```

Comment:
```suggestion
            return Optional.of(lockingCap);
```

This cannot be null since you are checking it above.

Reply:
It's `null` when there's no previous value in the storage.

Reply:
Ok, I see. So, maybe this method name could be changed `initializeLockingCap` to express clearly that it may not initialize the value if there's no previous value in the storage. Wdyt?

Reply:
We are keeping the same standard applied to the Whitelist Process Refactor. Please take a look at it:
https://github.com/rsksmart/rskj/blob/cda9adb5a5c396203c399a903e9ca422dd4a0943/rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistStorageProviderImpl.java#L62